### PR TITLE
Update the multi-ha docs

### PR DESCRIPTION
**Pull request by atovpeko**

### CODE REVIEW

File: use-timescale/ha-replicas/high-availability.md [hunk 1]
```diff
@@ -13,52 +13,59 @@ cloud_ui:
...
 
 ## What is HA replication?
 
-HA replicas are exact, up-to-date copies of your database that automatically take over operations as your primary data 
-node if the original primary data node becomes unavailable. HA replicas are synchronous or asynchronous hot standbys 
-hosted in multiple AWS availability zones(AZ) that use streaming replication to minimize the chance of data loss during 
-failover. That is, the primary node streams its write-ahead log (WAL) to the replicas.
+HA replicas are exact, up-to-date copies of your database hosted in multiple AWS availability zones (AZ) within the same region as your primary node. They automatically take over operations if the original primary data node becomes unavailable. The primary node streams its write-ahead log (WAL) to the replicas to minimize the chances of data loss during failover. 
+
+HA replicas can be synchronous and asynchronous.
```
Comment (billy-the-fish) [inline use-timescale/ha-replicas/high-availability.md:25]:
```suggestion
HA replicas can be:
```

File: use-timescale/ha-replicas/high-availability.md [hunk 2]
```diff
@@ -13,52 +13,59 @@ cloud_ui:
...
-node if the original primary data node becomes unavailable. HA replicas are synchronous or asynchronous hot standbys 
-hosted in multiple AWS availability zones(AZ) that use streaming replication to minimize the chance of data loss during 
-failover. That is, the primary node streams its write-ahead log (WAL) to the replicas.
+HA replicas are exact, up-to-date copies of your database hosted in multiple AWS availability zones (AZ) within the same region as your primary node. They automatically take over operations if the original primary data node becomes unavailable. The primary node streams its write-ahead log (WAL) to the replicas to minimize the chances of data loss during failover. 
+
+HA replicas can be synchronous and asynchronous. 
+
+- Synchronous replicas: the primary commits its next write once the replica confirms that the previous write is complete. There is no lag between the primary and the replica. They are in the same state at all times. This is preferable if you need the highest level of data integrity. However, this affects the primary ingestion time.
```
Comment (billy-the-fish) [inline use-timescale/ha-replicas/high-availability.md]:
```suggestion
- **Synchronous**: the primary commits its next write once the replica confirms that the previous write is complete. There is no lag between the primary and the replica. They are in the same state at all times. This is preferable if you need the highest level of data integrity. However, this affects the primary ingestion time.
```

File: use-timescale/ha-replicas/high-availability.md [hunk 3]
```diff
@@ -13,52 +13,59 @@ cloud_ui:
...
+  
+- Asynchronous replicas: the primary commits its next write without the confirmation of the previous write completion. The asynchronous HA replicas often have a lag, in both time and data, compared to the primary. This is preferable if you need the shortest primary ingest time.
```
Comment (billy-the-fish) [inline use-timescale/ha-replicas/high-availability.md]:
```suggestion
- **Asynchronous**: the primary commits its next write without the confirmation of the previous write completion. The asynchronous HA replicas often have a lag, in both time and data, compared to the primary. This is preferable if you need the shortest primary ingest time.
```

File: use-timescale/ha-replicas/high-availability.md [hunk 4]
```diff
@@ -13,52 +13,59 @@ cloud_ui:
...
+
+image from the doc draft
```
Comment (billy-the-fish) [inline use-timescale/ha-replicas/high-availability.md]: Can you add the image please. Just give it an imperfect name so we can delete it if necessary.

File: use-timescale/ha-replicas/high-availability.md [hunk 5]
```diff
@@ -13,52 +13,59 @@ cloud_ui:
...
-    always be in the exact same state as the primary, minimizing failover time and ensuring no data loss. However, 
-    synchronous replicas reduce ingest performance and do not provide a replica endpoint 
-     
-    Synchronous replication ensures the highest level of data consistency and safety. 
 
+- **High availability**: a single async replica in a different AWS availability zone from your primary. Provides high availability with cost efficiency. Best for production apps. 
+
+- **Highest availability**: two replicas in different AWS availability zones from your primary. Available replication modes are:
+
+  - **Optimized data performance** - two async replicas. Provides the highest level of availability (two AZs) and the ability to query the HA system. Best for absolutely critical apps.
```
Comment (billy-the-fish) [inline use-timescale/ha-replicas/high-availability.md]:
```suggestion
  - **Optimized data performance** - two async replicas. Provides the highest level of availability in two AZs, and the ability to query the HA system. Best for absolutely critical apps.
```

**Review by billy-the-fish — CHANGES_REQUESTED** [review]
Good doc, nearly there.

**Review by esclymeTS — DISMISSED** [review]
THIS IS SO GREAT. Just one small change: "Optimized data performance" to "High performance"

File: use-timescale/ha-replicas/high-availability.md (hunk 1 above)
Comment (atovpeko) [reply use-timescale/ha-replicas/high-availability.md:25]: To quote the style guide: Introduce a list with a complete sentence, not a partial one that's completed by the list items. You can also use _the following_ as a noun phrase. https://developers.google.com/style/lists

Am I to assume we are ignoring this rule to make docs less wordy?

File: use-timescale/ha-replicas/high-availability.md (hunk 2 above)
Comment (atovpeko) [reply use-timescale/ha-replicas/high-availability.md]: To quote the style guide: Use bold formatting, `<b>` or **, for [UI elements](https://developers.google.com/style/ui-elements#formatting) and at the beginning of [notices](https://developers.google.com/style/notices). Are we using bold more freely? I don't see why we would emphasize replica types here.

File: use-timescale/ha-replicas/high-availability.md (hunk 1 above)
Comment (billy-the-fish) [reply use-timescale/ha-replicas/high-availability.md:25]: Fair point.

File: use-timescale/ha-replicas/high-availability.md (hunk 2 above)
Comment (billy-the-fish) [reply use-timescale/ha-replicas/high-availability.md]: Fair point.

File: use-timescale/ha-replicas/high-availability.md (hunk 3 above)
Comment (billy-the-fish) [reply use-timescale/ha-replicas/high-availability.md]: Fair point.

**Review by billy-the-fish — APPROVED** [review]

**Pull request merged**